### PR TITLE
Streamlit model compare

### PR DESCRIPTION
**Pull request by jeremyjordan**
## What does this PR do?

Adds a new Streamlit app that lets us compare prediction distributions between two models.

### CODE REVIEW

File: flower_classifier/ui/model_compare.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+with col1:
+    st.write("Model A")
+    run_id, checkpoint_file = select_model("Model A")
+    if checkpoint_file:
+        model = components.download_model_wandb(run_id, checkpoint_file)
+        logits = components.make_prediction(model, pil_image)
+        preds = nn.functional.softmax(logits, dim=1)
+        top_pred = preds.max(1)
+        label = oxford_idx_to_names[top_pred.indices.item()]
```
Comment (huffmanjohnf) [inline flower_classifier/ui/model_compare.py]: Since this exact same code is used in col2 below (55-58) and in `app.py` (13-16), could we condense it into a function in `components.py` that inputs the model and pil_image and returns the preds and label?

**Review by huffmanjohnf — APPROVED** [review]
No issues, just general recommendation in `model_compare.py`

File: flower_classifier/ui/model_compare.py (hunk 1 above)
Comment (jeremyjordan) [reply flower_classifier/ui/model_compare.py]: good suggestion! i'll wrap `display_prediction` into a component
